ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.spec.ts

```diff
@@ -99,13 +98,28 @@ describe('FaqItemComponent', () => {
         expect(questionPanels.length).toBe(1);
     });
 
-    it('should call openJoinDialog', async () => {
-        // todo: fix this test
-        // component.accordion.openAll();
-        // fixture.detectChanges();
-        //
-        // const joinButton = fixture.debugElement.query(By.css('.join-btn')).nativeElement;
-        // joinButton.click();
-        // expect(communicationServiceSpy.openJoinDialog).toHaveBeenCalled();
+    it('should open join mailing list dialog', async () => {
```

Comment:
Do you need `async` in those refactored tests (here and below) ?

Reply:
removed, thanks!